### PR TITLE
Add Dropdown to select primary_zid in `edit profile` screen

### PR DESCRIPTION
**Pull request by ratik21**
### What does this do?

image

image

### CODE REVIEW

**Comment by ratik21** [comment]
Update: Added scrollbar
image

File: src/components/edit-profile/index.test.tsx [hunk 1]
```diff
@@ -94,11 +94,30 @@ describe('EditProfile', () => {
...
 
+  it('renders dropdown with ownedZIDs with the primaryZID as the first option', () => {
+    featureFlags.allowEditPrimaryZID = true;
+
+    const wrapper = subject({
+      currentPrimaryZID: '0://jane:smith',
+      ownedZIDs: ['0://john:doe', '0://jane:smith', '0://jijitsu:kaezen'],
+    });
+
+    const dropdown = wrapper.find('SelectInput[label=""]');
```
Comment (dalefukami) [inline src/components/edit-profile/index.test.tsx:105]: Is there more than one SelectInput at a time on this screen? I.E., do we need the `label` filter?

File: src/index.scss [hunk 2]
```diff
@@ -107,5 +107,5 @@ hr.glow:before {
 // and add a permanent node at the root level and set it as the popup container.
 // Using this method for now due to time constraints.
 [data-radix-popper-content-wrapper] {
-  z-index: 9999 !important;
+  z-index: 10001 !important;
```
Comment (dalefukami) [inline src/index.scss:110]: Is something else a higher z-index on our page?! ugggg

Comment (ratik21) [reply src/index.scss:110]: yeaaahhh i need this to display the "dropdown" on top the "modal" (which has 9999 z-index set if i remember correctly) 😢

File: src/components/edit-profile/index.test.tsx (hunk 1 above)
Comment (ratik21) [reply src/components/edit-profile/index.test.tsx:105]: we don't here, but i had to add it on the component side (`label = ''`), as it is a required field in the zUI component

**Pull request merged**